readme.md

```diff
@@ -3,8 +3,53 @@
 This package contains a parser for the Checker Framework's stub files:
 https://checkerframework.org/manual/#stub
 It is a fork of the [JavaParser](http://javaparser.org) project.
-The remainder of this README file is the JavaParser README.
 
+## Differences between the StubParser and JavaParser
+
+Following list is the differences that were added to the JavaParser to adjust it for using as Stubparser for [the Checker Framework](https://github.com/typetools/checker-framework).

```

Comment:
Minor: avoid long lines; break at 80 columns for consistency with the rest of the file.

Reply:
It would be nice to give a command line that shows the diffs between the StubParser and the upstream JavaParser.